### PR TITLE
Update MetaPhlAn2.9.12

### PR DESCRIPTION
**Pull request by fbeghini**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by fbeghini** [comment]
@BiocondaBot merge

File: recipes/metaphlan2/2.9.0/meta.yaml [hunk 1]
```diff
@@ -14,6 +14,10 @@ build:
   number: 0
   noarch: python
 
+extra:
+  watch:
+    enable: yes
+
```
Comment (dpryan79) [inline recipes/metaphlan2/2.9.0/meta.yaml]: Can you move the `extra:` section to the bottom of the file?

**Comment by dpryan79** [comment]
Thanks @fbeghini !

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**